README.md

```diff
@@ -13,6 +13,20 @@ setting for make to avoid using docker. Otherwise make sure to have these tools:
 - docker client and daemon
 - gnu make tools
 
+### Docker Container Setup
+
+We also provide a lightweight and easy way to test and run `oneview-golang`. The `hewlettpackardenterprise/hpe-oneview-sdk-for-golang:<tag>` docker image contains an installation of oneview-golang installation you can use by just pulling down the Docker Image:
+
+The Docker Store image `tag` consist of two sections: `<sdk_version-OV_version>`
+
+```bash
+# Download and store a local copy of hpe-oneview-sdk-for-golang and
+# use it as a Docker image.
+$ docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-golang:v1.4.0-OV5.2
+Run docker commands and this will in turn create
```

Comment:
its confusing to understand what it will create.

Reply:
Added the comment symbol to indicate that it is a comment.